tools/run_tests/artifacts/package_targets.py

```diff
@@ -142,18 +142,26 @@ def build_jobspec(self, inner_jobs=None):
 class PythonPackage:
     """Collects python eggs and wheels created in the artifact phase"""
 
-    def __init__(self, platform="", arch=""):
+    def __init__(self, platform="", arch="", run_in_arm64_job=False):
         self.name = "python_package"
         self.labels = ["package", "python", "linux"]
         self.platform = platform
         self.arch = arch
+        self.run_in_arm64_job = run_in_arm64_job
         if self.platform:
             self.labels.append(platform)
             self.name += "_" + platform
         if self.arch:
             self.labels.append(arch)
             self.name += "_" + arch
 
+            if arch == "noarch" and run_in_arm64_job:
+                # add a suffix as package target names must be unique
+                self.name += "_1"
+
+                self.labels.append("aarch64")
+                self.labels.append("exclude_in_collect_all_packages")
```

Comment:
Maybe we should name it something like "test_only_artifacts"? Anyway, "exclude_in_collect_all_packages" seems fine too.